### PR TITLE
Add Allnodes

### PR DESCRIPTION
**Pull request by 3eph1r0th**

### CODE REVIEW

File: akash/chain.json [hunk 1]
```diff
@@ -79,6 +79,16 @@
       {
         "id": "47f7b7a021497ad7a338ea041f19a1a11ae06795",
         "address": "rpc.la.akash.farm:26656"
+      },
+      {
+        "id": "e1b058e5cfa2b836ddaa496b10911da62dcf182e",
+        "address": "akash-seed-1.allnodes.me:26656"
```
Comment (JeremyParish69) [inline akash/chain.json:85]: Comma missing after 'address' in every file

Comment (3eph1r0th) [reply akash/chain.json]: Thank you. Should be fixed now

File: cryptoorgchain/chain.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{
+м{
```
Comment (JeremyParish69) [inline cryptoorgchain/chain.json:1]: this M here is causing the failure

Comment (3eph1r0th) [reply cryptoorgchain/chain.json]: Thank you. Fixed as well

**Pull request merged**